SU-SWS/stanford_paragraph_types
# Earth-15 Column Text Paragraph Type

# READY FOR REVIEW

# Summary
- Added column text paragraph type

# Needed By (Date)
- End of sprint

# Urgency
- low

# Steps to Test

1. Checkout this branch and same branch in [stanford_components](https://github.com/SU-SWS/stanford_components/pull/72) & [SE3_BLT](https://github.com/stanford-earth/SE3_BLT/pull/38)
2. config import
3. clear cache
4. create complex page with "Column Text" component.
5. change variants to check its display.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_components/pull/72
- https://github.com/stanford-earth/SE3_BLT/pull/38

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
